src/h5web/providers/jupyter/devApi.ts

```diff
@@ -0,0 +1,10 @@
+import { JupyterApi } from './api';
+
+export class JupyterDevApi extends JupyterApi {
+  /* API compatible with jupyterlab_hdf v0.5.1 */
+  public constructor(url: string, filepath: string) {
+    super(url, filepath);
+    // eslint-disable-next-line no-console
+    console.log('Using Jupyter dev API');
```

Comment:
I thought this could be handy for debugging but I don't mind removing it

Reply:
Good call, it will save us a lot of time when debugging.

Reply:
I replaced `log` with `warn` to make it stand-out among profiling data

Reply:
Hmm, I would have suggested `debug` instead -- it doesn't stand out, but you can filter it in the dev tools.

Reply:
hmm no you're right -- if we're using the dev provider in a prod environment, we should see it clearly